csuntechlab/takeoff
# TKO-34 -- FE: Implement form validation for the student profile setup

# Description
- Added form validation for the profile and account setup.
- Profile photo's accepted file types are now restricted to jpeg jpg and png.
- The profile inputs did not need validation as they are completely optional and can be whatever the user decides.
  
# Testing
- Navigate to `/account-setup`
- Try to enter bad inputs and verify the form validation does its thing
- Navigate to `/profile-setup`
- Make sure profile photo upload only allows jpg, jpeg, png

- Console.log is left for later when the forms will actually be submitted

# Installation
- Possibly `yarn install`
